mountpoint-s3/tests/cli.rs

```diff
@@ -301,3 +301,42 @@ fn verify_new_part_size_config_conflict_with_old_one(
 
     Ok(())
 }
+
+#[cfg(feature = "fstab")]
```

Comment:
Should we add some end-to-end tests that integrates with real fstab? Maybe we can spawn a container that configures Mountpoint using fstab or systemd.

Reply:
I'd want to do that in a separate PR as that would be quite a lot of additional effort on top of what's going on here